src/renderer/Instance.ts

```diff
@@ -114,16 +114,15 @@ export default class Instance implements InstanceData {
 			return proc;
 		}
 	}
-	public install(): Promise<TaskRuntime<Task.State>> {
-		return new Promise((resolve, reject) => {
+	public async install(): Promise<TaskRuntime<Task.State>> {
+		return await (new Promise((resolve, reject) => {
 			this.isInstalling = true;
 			const location: MinecraftLocation = MinecraftFolder.from(path.join(app.getPath("userData"), "./game/"));
 			console.log(`Starting installation of instance "${this.name}" with version "${this.id}" into dir "${location.root}"`);
-			// const res = await Installer.install("client", this, location);
 
 			const installTask: Task<ResolvedVersion> = Installer.installTask("client", this, location);
-			const taskProgress: TaskProgress | null = document.getElementById("task-progress") as TaskProgress;
-			const runtime = taskProgress?.addInstallTask(installTask, this.name);
+			const taskProgress = document.querySelector<TaskProgress>("task-progress");
+			const runtime = taskProgress!.addInstallTask(installTask, this.name);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This assertion is unnecessary since it does not change the type of the expression.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5396733)